flow/templates/print_status.jinja

```diff
@@ -0,0 +1,64 @@
+{# The following variables are available to all scripts. #}
+{% if overview %}
+{% set format_progress_bar = ("%%-%ss  %%-%ss" | format(column_width_label,column_width_bar,)) %}
+{% endif %}
+{% if detailed %}
+{% set alias = scheduler_status_code %}
+{% set column_width_operation = [column_width_operation,5] | max %}
+{% set format_job_id = ("%%-%ss  " | format(column_width_id,)) %}
+{% set format_operation = ("%%-%ss %%3s  " | format(column_width_operation,)) %}
+{% set format_operation_title = ("%%-%ss  " | format(column_width_operation+4,)) %}
+{% set format_label = ("%%-%ss" | format(column_width_total_label,)) %}
+{% set ns = namespace(dash='', format_parameters='') %}
+{% if parameters %}
+{% for para in parameters %}
+{% set ns.dash = ns.dash + ("%s  " | format('-' * column_width_parameters[loop.index-1])) %}
```

Comment:
Use `loop.index0` instead of `loop.index-1`. Documented here: http://jinja.pocoo.org/docs/2.10/templates/#for